### ISSUE TITLE
301 Moved Permanently on Web Service

### ISSUE THREAD

**Issue by LeobardoRomero** [inukshuk/anystyle#123]
I use:

http://anystyle.io/parse/references

but:

object(HttpSocketResponse) {
	body => '<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.12.1</center>
</body>
</html>
'
	headers => array(
		'Server' => 'nginx/1.12.1',
		'Date' => 'Tue, 13 Aug 2019 23:23:42 GMT',
		'Content-Type' => 'text/html',
		'Content-Length' => '185',
		'Connection' => 'close',
		'Location' => 'https://anystyle.io/parse/references'
	)
	cookies => false
	httpVersion => 'HTTP/1.1'
	code => '301'
	reasonPhrase => 'Moved Permanently'
	raw => 'HTTP/1.1 301 Moved Permanently
Server: nginx/1.12.1
Date: Tue, 13 Aug 2019 23:23:42 GMT
Content-Type: text/html
Content-Length: 185
Connection: close
Location: https://anystyle.io/parse/references

<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.12.1</center>
</body>
</html>
'
	context => array()
}

**Comment by inukshuk** [comment]
We're in the process of shutting down (or moving) the anystyle.io web service. For the time being, I'd urge you to use the `anystyle-cli` tool instead: since version 1.0 it is much easier to install and it will give you much better and stable results compared to the web service.

**Comment by nchachereau** [comment]
it seems that the web service has been resurrected (761521efbd243596d75801154269226e8b0afb06), which is excellent news for many users. Is it there to stay?

(We reported briefly about the shutdown on [one of our last blog posts](https://zotero.hypotheses.org/2939#anystyle) on Le blog Zotero francophone. We'd be happy to report that the service is up again!)

**Comment by inukshuk** [comment]
Yes! The web-service has been updated to use the latest version of the parser and lives happily on a new server. The web app itself [is now also open source](https://github.com/inukshuk/anystyle.io) so it's definitely here to stay: even if something happens to anystyle.io, anyone can just it on their own public or private server.

**Comment by nchachereau** [comment]
That's excellent news, thank you!

**Comment by urspx** [comment]
Wow, amazing news!! V. glad to hear, thank you 💫
